docs/R-IOC/r07_data_import_export.md

```diff
@@ -0,0 +1,222 @@
+## Data Import and Export
+
+We can deal with different formats of data with R, such as
+the text files (.csv, .tsv, .txt), the Excel files (.xlx, .xlsx), and the R data file formats (.RDS, .RData).
+It's also possible to read or write other program-speficied formats, for example
+"SAS", "SPSS" or "Minitab" files.
+
+
+## Importing Data
+
+In order to avoid error when you import the file,
+it's suggested to name the data columns following the naming conventions (see [Best Practices](r04_bestpractices.md) section).
+
+### Read Text Files (.csv/.tsv/.txt)
+
+With the functions from the basic package "utils":
+
+```r
+## read a comma separated file ".csv"
+my_csv <- read.csv(
+  file = "path/to/my_file.csv",
+  header = TRUE,    # whether the file has a colnames in the 1st row
+  sep = ",",        # what is the field separator
+  quote = "\"",     # the character(s) for quotes
+  dec = ".",        # the character for decimal points
+  fill = TRUE,      # in case of rows with unequal length, whether to add empty field
+  comment.char = "" # character used to indicated rows as comment lines, use empty string to turn off the interpretation of comment lines
+)
+
+## read a tab separated file ".tsv"
+my_delim <- read.delim(
+  file = "path/to/my_file.tsv",
+  header = TRUE,
+  sep = "\t",       # what is the field separator
+  quote = "\"",
+  dec = ".",
+  fill = TRUE,
+  comment.char = ""
+)
+
+## read a text file ".txt"
```

Comment:
Apart the fact from `read.table` is buggy and reserves you nasty nasty bugs (you are now warned !),
in this exemple, the `sep = ""` is a bit odd, isn't it ?

Reply:
ok, now with sep = "," it is better !

I think also that the sep parameter does not specifically deserve a comment (# what is the field separator). I would clear off this comment

Reply:
Agreed `read.table` is not super cool to use, but as `read.csv` and `read.delim` are actually based on it, I would introduce this function anyway. `sep =""` is the default value from the doc, now changed to `sep =" "` in the example.